src/selectors/metadata.js

```diff
@@ -24,6 +23,18 @@ const runCommandTemplates = {
   task: (name) => `kedro run --to-nodes ${name}`,
 };
 
+/**
+ * Unwrap parameters by extracting the first object
+ *  @param {?object} parameters The parameters
```

Comment:
I assumed this refers to the raw parameter object returned from the server?

Reply:
yes